internal/coremsgs/en_config_descriptions.go

```diff
@@ -185,6 +185,30 @@ var (
+	ConfigPluginBlockchainTezosTezosconnectBatchSize                   = ffc("config.plugins.blockchain[].tezos.tezosconnect.batchSize", "The number of events Tezosconnect should batch together for delivery to FireFly core. Only applies when automatically creating a new event stream", i18n.IntType)
+	ConfigPluginBlockchainTezosTezosconnectBatchTimeout                = ffc("config.plugins.blockchain[].tezos.tezosconnect.batchTimeout", "How long Tezosconnect should wait for new events to arrive and fill a batch, before sending the batch to FireFly core. Only applies when automatically creating a new event stream", i18n.TimeDurationType)
+	ConfigPluginBlockchainTezosTezosconnectInstance                    = ffc("config.plugins.blockchain[].tezos.tezosconnect.instance", "The Tezosconnect address of the FireFly BatchPin smart contract that has been deployed to the blockchain", "Address "+i18n.StringType)
+	ConfigPluginBlockchainTezosTezosconnectFromBlock                   = ffc("config.plugins.blockchain[].tezos.tezosconnect.fromBlock", "The first event this FireFly instance should listen to from the BatchPin smart contract. Default=0. Only affects initial creation of the event stream", "Address "+i18n.StringType)
+	ConfigPluginBlockchainTezosTezosconnectPrefixLong                  = ffc("config.plugins.blockchain[].tezos.tezosconnect.prefixLong", "The prefix that will be used for Tezosconnect specific HTTP headers when FireFly makes requests to Tezosconnect", i18n.StringType)
+	ConfigPluginBlockchainTezosTezosconnectPrefixShort                 = ffc("config.plugins.blockchain[].tezos.tezosconnect.prefixShort", "The prefix that will be used for Tezosconnect specific query parameters when FireFly makes requests to Tezosconnect", i18n.StringType)
+	ConfigPluginBlockchainTezosTezosconnectTopic                       = ffc("config.plugins.blockchain[].tezos.tezosconnect.topic", "The websocket listen topic that the node should register on, which is important if there are multiple nodes using a single tezosconnect", i18n.StringType)
+	ConfigPluginBlockchainTezosTezosconnectURL                         = ffc("config.plugins.blockchain[].tezos.tezosconnect.url", "The URL of the Tezosconnect instance", "URL "+i18n.StringType)
+	ConfigPluginBlockchainTezosTezosconnectProxyURL                    = ffc("config.plugins.blockchain[].tezos.tezosconnect.proxy.url", "Optional HTTP proxy server to use when connecting to Tezosconnect", "URL "+i18n.StringType)
+
+	ConfigPluginBlockchainTezosFFTMURL      = ffc("config.plugins.blockchain[].tezos.fftm.url", "The URL of the FireFly Transaction Manager runtime, if enabled", i18n.StringType)
+	ConfigPluginBlockchainTezosFFTMProxyURL = ffc("config.plugins.blockchain[].tezos.fftm.proxy.url", "Optional HTTP proxy server to use when connecting to the Transaction Manager", i18n.StringType)
```

Comment:
I think the equivalent config parameters for Ethereum have been removed. I am pretty sure they were deprecated after a very early stage build of FFTM (when it was its own runtime, rather than a library). @peterbroadhurst Do you see any reason to keep these options, or the equivalent Ethereum options?